Database/Constraint/AddTicket Procedure.sql

```diff
@@ -0,0 +1,50 @@
+﻿DROP PROCEDURE if exists sp_valid_ticket
+GO
+CREATE PROCEDURE sp_valid_ticket
```

Comment:
Is dit geen betere naam? - sp_add_ticket_type

Reply:
nee (grapje ja)